### ISSUE TITLE
Feature: Suppress 204 Logs from Gateway

### ISSUE THREAD

**Issue by MikePresman** [grafana/loki#8963]
**Is your feature request related to a problem? Please describe.**
I'm frustrated when I see thousands of requests akin to the tune of `204 "POST /loki/api/v1/push HTTP/1.1"`. They don't help anyone and should be suppressed.

**Describe the solution you'd like**
A flag to disable these logs from coming out of gateway.

**Describe alternatives you've considered**
Looked through the code to enable log level but not sure how loki handles args to suppress logs.

**Additional context**

**Comment by slim-bean** [comment]
yeah.... unfortunately this isn't as easy as I wish it was as I believe these logs come from a library we used to run the http server components, and debug level is an all or nothing in that library for logging requests 😬 

I think this is a totally reasonable request I just wish it was easy to do...

**Comment by calmon-mms** [comment]
Is there any update related to this topic?

**Comment by winem** [comment]
I can't tell exactly since when I don't see them anymore but at least with Loki 3.0 they do not appear any longer in the gateway logs.

**Comment by JohnFrampton** [comment]
Hi, 
I have several millions of the log lines every hour which is really covering all other necessary information in the log. I would also be very happy, if there would be a way to cut out all 2** er messages from the gateway

**Comment by ManuelCortes23** [comment]
Any update?

**Comment by KarBroden** [comment]
I found the option verboseLogging on the gateway service. Setting that option to false got rid of 2xx log lines.

**Comment by JStickler** [comment]
Closing this issue as already built. Setting `gateway.verboseLogging: false` in the Helm chart values suppresses 2xx log lines from the gateway. This was confirmed by a commenter on this issue.
